rust-toolchain

```diff
@@ -1 +1 @@
-nightly-2022-01-02
\ No newline at end of file
+nightly-2021-12-15
```

Comment:
due to https://github.com/bytecodealliance/rustix/issues/142 ; once wasmtime bumps up its dependencies we can revert this.